### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by magicansk**
Spelling fixed

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-curl the next few years - perhaps
```
Comment (bagder) [inline docs/ROADMAP.md]: I don't think `curl` should be capitalized, even when starting a sentence

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -91,28 +91,28 @@ New stuff - curl
...
+   don't have to be constructed before calling curl.
 
 Improve
 -------
 
-1. build for windows (considered hard by many users)
+1. Build for windows (considered hard by many users).
 
-2. curl -h output (considered overwhelming to users)
+2. Curl -h output (considered overwhelming to users).
```
Comment (bagder) [inline docs/ROADMAP.md]: ... and certainly not when used in a command line snippet like this.

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -91,28 +91,28 @@ New stuff - curl
...
    definitions and such like that are currently earmarked to be removed years
-   ago
+   ago.
 
 Remove
 ------
 
-1. makefile.vc files as there is no point in maintaining two sets of Windows
-   makefiles. Note: These are currently being used by the Windows autobuilds
+1. Makefile.vc files as there is no point in maintaining two sets of Windows
```
Comment (bagder) [inline docs/ROADMAP.md]: This is a file name and they are often case sensitive so this cannot be capitalized without changing meaning.

Comment (magicansk) [reply docs/ROADMAP.md]: fixed.

File: docs/ROADMAP.md (hunk 2 above)
Comment (magicansk) [reply docs/ROADMAP.md]: fixed.

File: docs/ROADMAP.md (hunk 1 above)
Comment (magicansk) [reply docs/ROADMAP.md]: Nice to know, thanks and fixed.

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**